main.cpp

```diff
@@ -1,16 +1,69 @@
 #include "main.h"
 #include "rc.h"
 #include <filesystem>
+#include <iostream>
+
+using namespace Gdiplus;
+using namespace std;
+
+main_window::main_window() {
+	TCHAR file[MAX_PATH] = {};
+
+	ofn.lStructSize = sizeof(ofn);
+	ofn.hwndOwner = *this;
```

Comment:
Ovo je krivo (i sruši program) barem iz dva razloga:
- u konstruktoru klase `main_window` još ne postoji `HWND` (pa `*this` nema smisla)
- pointer `lpstrFile` pokazuje na lokalno polje (koje poslije ne postoji)

Prebacite `ofn` kao lokalnu varijablu unutar `case`-a unutar `on_command`.